vite-config/manifest.ts

```diff
@@ -7,53 +12,53 @@ export const manifest: Partial<ManifestOptions> = {
     'ADAMANT is a decentralized anonymous messenger based on the blockchain system. It’s independent of any governments or corporations, and even developers due to the distributed network infrastructure that contains an open-source code.',
   icons: [
     {
-      src: '/img/icons/android-chrome-36x36.png',
+      src: joinUrl(BASE, 'img/icons/android-chrome-36x36.png'),
```

Comment:
Why not using `import.meta.env.BASE_URL`?

Reply:
He is caching it in the variable

Reply:
Is that the case? @S-FrontendDev

Reply:
If yes, just resolve the conversation